src/libraries/System.Resources.Extensions/src/BinaryFormat/Deserializer/ClassRecordFieldInfoDeserializer.cs

```diff
@@ -26,7 +26,7 @@ internal sealed class ClassRecordFieldInfoDeserializer : ClassRecordDeserializer
     internal ClassRecordFieldInfoDeserializer(
         Runtime.Serialization.BinaryFormat.ClassRecord classRecord,
         object @object,
-        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.NonPublicFields | DynamicallyAccessedMemberTypes.PublicFields)]
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
```

Comment:
Thanks, missed this one as it apparently needs all to be able to look at base class fields.